.github/workflows/ci.yml

```diff
@@ -117,17 +117,18 @@ jobs:
         const fs = require("fs");
         const workspacePkgJsonPath = "./package.json";
         const workspacePkgJson = require(workspacePkgJsonPath);
-        workspacePkgJson.pnpm.overrides = {
-          ...workspacePkgJson.pnpm.overrides,
```

Comment:
Nope, I should have left that out.

Reply:
Oh that is the old code...  
indecisive. We shouldn't be using overrides.. okay I'll bite..

Reply:
Since I've removed the `pnpm` object, I'm spreading both that and `pnpm.overrides`. This whole deal should be revisited. As a comment above states, it is definitely hacky.

Reply:
thanks